### PR TITLE
Shows Merge

### PR DESCRIPTION
**Pull request by lerisse**
Had to remove 

          <div>
            <Shows review_id={id} />
          </div>

from Reviews card commons to fix loop

### CODE REVIEW

File: client/src/components/commons/ReviewCard.jsx [hunk 1]
```diff
@@ -26,6 +25,7 @@ const {
   commentTime,
   textTime,
 } = styles;
+import Shows from './../showsView/Shows';
```
Comment (AndrewHuang123) [inline client/src/components/commons/ReviewCard.jsx:28]: Do not import another component if it's not being used in the current file.

File: client/src/components/showsView/AddComment.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+//             />
+//           </div>
+
+//           <input type="submit" value="Submit Comment" />
+//         </div>
+//       </form>
+//     );
+//   }
+// }
+// export default AddComment;
```
Comment (AndrewHuang123) [inline client/src/components/showsView/AddComment.jsx:55]: This whole file is commented out. Probably best to just delete the file.

File: client/src/components/showsView/AddReview.jsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+// import { useAuth } from '../../contexts/AuthContext';
+
+// const { currentUser } = useAuth();
```
Comment (AndrewHuang123) [inline client/src/components/showsView/AddReview.jsx:4]: Removed commented section.

File: client/src/components/showsView/AddReview.jsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {
+      show_rating: '',
+      text: '',
+    };
+  }
+
+  handleInputChange(e) {
+    // console.log("THIS IS THE NAME", selectedPokemon.name);
```
Comment (AndrewHuang123) [inline client/src/components/showsView/AddReview.jsx:16]: Remove comment.

File: client/src/components/showsView/AddReview.jsx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    this.setState({ [e.target.name]: e.target.value });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+
+    // console.log("THIS IS NAME OF POKEMON FROM FORM", this.state.name);
```
Comment (AndrewHuang123) [inline client/src/components/showsView/AddReview.jsx:23]: Remove comment.

File: client/src/components/showsView/AddReview.jsx [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+  render() {
+    return (
+      <form onSubmit={this.handleSubmit.bind(this)}>
+        <div>
+          <div className="addReviewBox">
+            Write Review:
+            <input
+              type="text"
+              // addReview="addReview"
```
Comment (AndrewHuang123) [inline client/src/components/showsView/AddReview.jsx:45]: Remove comment.

File: client/src/components/commons/ReviewCard.jsx (hunk 1 above)
Comment (WarrenWongCodes) [reply client/src/components/commons/ReviewCard.jsx:28]: removed

**Comment by lerisse** [comment]
Closed due to duplication, these changes were brought over to header-component, which is now merged with dev

**Pull request closed without merging**